app/src/main/res/layout/content_main.xml

```diff
@@ -22,26 +22,24 @@
         android:id="@+id/Recycleview"
         android:layout_width="0dp"
         android:layout_height="0dp"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toTopOf="@+id/adView"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toTopOf="parent" />
 
 
-    <com.google.android.gms.ads.AdView xmlns:ads="http://schemas.android.com/apk/res-auto"
+    <com.google.android.gms.ads.AdView
         android:id="@+id/adView"
-        android:layout_width="wrap_content"
+        android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
         android:layout_centerHorizontal="true"
-        android:layout_marginStart="20dp"
-        android:layout_marginTop="461dp"
         android:foregroundGravity="bottom"
-        ads:adSize="BANNER"
-        ads:adUnitId="ca-app-pub-7250174976428336/1382229664"
-        ads:layout_constraintBottom_toTopOf="@+id/fragment_container"
-        ads:layout_constraintStart_toStartOf="parent"
-        ads:layout_constraintTop_toTopOf="parent"></com.google.android.gms.ads.AdView>
+        app:adSize="BANNER"
+        app:adUnitId="ca-app-pub-7250174976428336/1382229664"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent" />
 
 
 </android.support.constraint.ConstraintLayout>
```

Comment:
add a new line to the bottom

Reply:
Please resolove the conflicts

Reply:
Where is conflit ?

Reply:
could you pls see the bottom of the PR and resolve the conflicts 
![image](https://user-images.githubusercontent.com/37568816/139522701-c30f6ffa-486f-4564-ab3a-4e30cb5d4376.png)